layouts/joomla/content/readmore.php

```diff
@@ -38,9 +38,11 @@
 			<?php echo Text::_('JGLOBAL_READ_MORE'); ?>
 		</a>
 	<?php else : ?>
-		<a class="btn btn-secondary" href="<?php echo $displayData['link']; ?>" aria-label="<?php echo Text::sprintf('JGLOBAL_READ_MORE_TITLE', $this->escape($item->title)); ?>">
+		<?php $text = Text::sprintf('JGLOBAL_READ_MORE_TITLE', HTMLHelper::_('string.truncate', $item->title, $params->get('readmore_limit'))); ?>
```

Comment:
```suggestion
		<?php $text = Text::sprintf('JGLOBAL_READ_MORE_TITLE', $this->escape(HTMLHelper::_('string.truncate', $item->title, $params->get('readmore_limit')))); ?>
```
As suggested by Phil